doc/gov/chapters/chapter02.md

```diff
@@ -34,13 +34,8 @@ As an open-source community organization, CNTT has developed the following proce
   
   - **What happens once I get onboarded?** Once you have reviewed the CNTT onboarding information and have agreed to the perquisite terms and agreements associated with participating and contributing to the Task Force, and you have completed onboarding and now it will be your opportunity to join desired work streams. For an initial period after onboarding, you will be assigned a volunteer mentor from the Task Force to help you get up to speed and contributing quickly.
```

Comment:
Should the description of what happens once "I have onboarded" include the onboarding steps or simply state, "Once you have completed the onboarding process, you will be able to join work streams. If you want to retain current wording, you still need to change (after all the "and"s) the phrase, "... and now it will be your opportunity to join desired work streams" to, say, "...  it will be your opportunity to join desired work streams."

Typo, "perquisite" ---> "prerequisite"

Reply:
Pankaj. This is the getting started. It is not the onboarding packet itself. But point taken. I would like to get all draft versions of doc in first before we start dickering over fine points.